stdlib/public/core/ReflectionMirror.swift

```diff
@@ -12,8 +12,9 @@
 
 import SwiftShims
 
-@_silgen_name("swift_isClassType")
-internal func _isClassType(_: Any.Type) -> Bool
+internal func _isClassType(_ type: Any.Type) -> Bool {
+  return swift_isClassType(unsafeBitCast(type, to: UnsafeRawPointer.self))
```

Comment:
I don't understand this.  I guess I should look at the implementation.  But this deserves a comment.  You are passing the metatype structure as a pointer?

Reply:
According to here, a thick metatype is represented with a metadata pointer.
 https://github.com/apple/swift/blob/dd32711a5f5c5bd28a6401fc3ede1a0bf344b49e/lib/IRGen/GenType.cpp#L2464-L2466

So the `unsafeBitCast` is a safe operation. But I agree this is not clear to everyone, so it should be described in comments.